docs/roadmap/STATUS.json

```
@@ -190,22 +190,22 @@
     {
       "id": "cicd-signal-deltas-2026",
-      "status": "in_progress",
+      "status": "complete",
       "owner": "codex",
       "notes": "Documented CI/CD high-signal deltas with enforced action register and evidence targets."
     }
   ],
   "summary": {
     "rc_ready": 8,
-    "partial": 2,
+    "complete": 2,
```

Biome (2.4.4)

[error] The key complete was already declared.

(lint/suspicious/noDuplicateObjectKeys)